bin/delete-stub-files

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+for dir in exercises/*; do
+  stub=$(echo $(basename $dir) | tr - _)
+  file="$dir/$stub.go"
+  if [ -f $file ]; then
+    echo "deleting stub $file"
```

Comment:
verbosity so that if ever a file is removed that shouldn't be, or a file isn't removed that should be, looking at travis logs can help diagnose. I think it's better to have this than not, but would like to point it out in case someone disagree